README.md

```diff
@@ -428,6 +428,48 @@ in Step 1 of the
 [Set up a Conjur OSS environment with persistence](#set-up-a-conjur-oss-environment-with-persistence)
 section above.
 
+### Adding or Modifying Container Environment Variables
+
+This section describes the process of either adding or modifying  environment variables in a running container. This process only requires stopping the container in question for a short time, while the rest of the system continues running.
```

Comment:
You'll want to split this into multiple lines, we try to wrap at around 100 characters or so.  An easy trick to remember - you'll notice if a line break doesn't exist if there is no line number next to your line.

There are a few other places in the doc where you'll wanna be sure to do this.